BeFake/__main__.py

```diff
@@ -131,10 +131,10 @@ def feed(bf, feed_id, save_location, realmoji_location, instant_realmoji_locatio
 
     for item in feed:
         if feed_id == "memories":
-            click.echo("saving memory", item.memory_day)
+            click.echo(f"saving memory {}".format(item.memory_day))
```

Comment:
```suggestion
            click.echo("saving memory {}".format(item.memory_day))
```

`f-string: empty expression not allowed (__main__.py, line 134)`

Reply:
> This gives `f-string: empty expression not allowed (__main__.py, line 134)`

With the suggestion @NicoWeio made the code seems to be working fine 👍